config/locales/en.yml

```diff
@@ -54,6 +54,7 @@ en:
       profile_not_found: >
         You are using a non-existent profile called '%{profile}',
         please check your technologies.
+      invalid_technologies: "The LES contains the following invalid technologies: %{invalid_technologies}. This will result in an incomplete network calculation."
```

Comment:
Could this be more descriptive? "[something] is invalid" doesn't tell the user what is actually wrong, nor how to fix it. For example:

--

This LES contains the following invalid technologies:

* The profile used by [technology] on [endpoint] no longer exists. Please assign a new profile.

--

This could get quite long; perhaps testing_grounds/show could show a very simple "This LES contains invalid technologies; to remedy this, please edit the LES (link to edit page)." ... and show the full error message on the edit page?